pyftpdlib/test/functional_ssl_client_certfile_tests.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2007-2016 Giampaolo Rodola' <g.rodola@gmail.com>.
+# Use of this source code is governed by MIT license that can be
+# found in the LICENSE file.
+#
+#
+# Does not follow naming convention of other tests because this
+# CANNOT be run in the same test suite with test_functional_ssl.
+# The test parallelism causes SSL errors when there should be none
+# Please run these tests separately
```

Comment:
Thanks for reminding. I found out the root cause is `TLS_FTPHandler.get_ssl_context` is a static method, thus it would get messed up in test cases. I changed it to non-static and the issue is gone, so these tests are merged back to `test_functional_ssl.py`.